src/tacacs/pam/Makefile

```diff
@@ -20,6 +20,7 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	git apply ../0004-management-vrf-support.patch
 	git apply ../0005-pam-Modify-parsing-of-IP-address-and-port-number-to-.patch
 	git apply ../0006-Add-support-for-source-ip-address.patch
+	git apply ../0007-Add-TACACS-support-library-and-fix-memory-leak-issue.patch
```

Comment:
>0007 [](http://example.com/codeflow?start=14&length=4)

 does it make sense to have 2 patches?

Reply:
This issue fixed in another PR: https://github.com/Azure/sonic-buildimage/pull/8659
After that PR complete first, I will merge this PR.

Reply:
Fixed.